misc/cleanup-dash.py

```diff
@@ -144,6 +143,25 @@ def drop_some_internal_elements(dashboard):
             del dashboard['__inputs']
         if '__requires' in element:
             del dashboard['__requires']
+        if 'panels' in element:
+            for panel_index, panel in enumerate(dashboard['panels']):
+                if 'datasource' in panel:
+                    if panel['datasource'] == "${DS_PROMETHEUS}":
```

Comment:
This is correct technically, but some conventions does not encourage mix single and double quotes for simple strings.

Reply:
Thx, quotes have been replaced.